llvm/docs/GlobalISel/GenericOpcode.rst

```diff
@@ -725,7 +725,13 @@ Mixing scalable vectors and fixed vectors are not allowed.
 G_CONCAT_VECTORS
 ^^^^^^^^^^^^^^^^
 
-Concatenate two vectors to form a longer vector.
+Concatenate vectors to form a longer vector.
+
+.. code-block:: none
+
+  %4:_(<16 x i32>) = G_CONCAT_VECTORS %3:_(<4 x i32>), %2:_(<4 x i32>),
+                                      %1:_(<4 x i32>), %0:_(<4 x i32>)
```

Comment:
```suggestion
  %4:_(<16 x i32>) = G_CONCAT_VECTORS %0:_(<4 x i32>), %1:_(<4 x i32>),
                                      %2:_(<4 x i32>), %3:_(<4 x i32>)
```

Better use in order numbers to avoid confusing about the element indexing